.circleci/config.yml

```diff
@@ -69,9 +69,15 @@ jobs:
           name: standup and test audius
           command: |
             cd ~/dot-slash-audius
+            REPO_OWNER="AudiusProject"
+            REPO_NAME="audius-docker-compose"
+            BRANCH_NAME="stage"
+            COMPOSE_COMMIT_SHA=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+              "https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/branches/$BRANCH_NAME" | \
+              jq -r '.commit.sha')
```

Comment:
this was needed since I couldn't access the `COMPOSE_COMMIT_SHA` from the build workflow